dev/devicelab/lib/tasks/integration_tests.dart

```diff
@@ -44,10 +44,11 @@ TaskFunction createExternalUiIntegrationTest() {
   );
 }
 
-TaskFunction createPlatformChannelSampleTest() {
+TaskFunction createPlatformChannelSampleTest({String? deviceIdOverride}) {
```

Comment:
This and the change below follow the pattern from https://github.com/flutter/flutter/pull/45264